algosdk/__init__.pyi

```diff
@@ -0,0 +1,68 @@
+## File generated from scripts/generate_init.py.
```

Comment:
Is there an advantage in checking in the `.pyi` file into the repo?

Reply:
From my own usage experience, to benefit from these, the `pip installing` project only benefits under this condition. But maybe there's a way to add this into the build step and not check in?

Reply:
Following up from standup, @jasonpaulos made the point that it'll be a little easier in our release process if we _don't_ add the new step of running `generate_init.py` to create an artifact `__init__.pyi` which will only exist in the release. I'm convinced of this point but am open to further discussion.

Reply:
I think it makes sense to include it to simplify the release process - resolving 👍